README.md

```diff
@@ -2,11 +2,30 @@
 
 ## Installation
 
-To run the tests you need to install the emaj database extension.
-Follow the instructions on [emaj website](http://emaj.readthedocs.io/en/latest/download.html). There may be some problems so the step by step installation, which worked for us, is listed below.
-
-* Download the package from [PGXN](https://pgxn.org/dist/e-maj/) (see instructions in emaj docu) and unzip it
-* Copy latest emaj version sql file (e.g. `emaj--2.0.1.sql` in unzipped folder) and `emaj.control` file in the `SHAREDIR` directory (check with `pg_config --sharedir`). On e.g. Arch Linux it's `/usr/share/postgresql`, maybe you have to copy the files to `/usr/share/postgresql/extension`. You can leave the `emaj--2.0.1.sql` file where it is and just adjust the directory parameter in the `emaj.control` file to reflect the location of it. Skip if you copied both files.
-* Rest of the steps will be executed when you run `cucumber` (see `features/support/emaj.sql`)
-* First execution could take a while (because there is a `git clone` etc.), everything is fine if all the tests are green in the end
-* If you want the tests to run on specific commits you can run `ONTOHUB_BACKEND_VERSION='12345678abcdefg' ONTOHUB_FRONTED_VERSION='12345678abcdefg' HETS_RABBITMQ_WRAPPER_VERSION='12345678abcdefg' cucumber`. If you want e.g. just the backend, you only use `ONTOHUB_BACKEND_VERSION` and so on. Remember that the tests may fail depending on the commits, you check out.
\ No newline at end of file
+To run the tests you need to install the emaj database extension. This extensions enables rollbacks on the database without using database-level transactions.
+If the following instructions don't work, follow the instructions on the [emaj Github Readme](https://github.com/beaud76/emaj) or the [emaj documentation](http://emaj.readthedocs.io/en/latest/install.html).
+
+Go to the [extension download page](https://pgxn.org/dist/e-maj/) and find the version number of the latest release of E-Maj.
+Run the following commands in your bash or zsh (and adjust the version in the first line):
+
+```bash
```

Comment:
Perhaps create a script in this repo that does these steps? Would make it easier if we want to update to a newer version of emaj.

Reply:
I thought about that, too, but such a script would fail when the URLs change and one needs to specify the version when calling the script... And you need to open the page to find the current version number.